spec/spec_helper.rb

```diff
@@ -1,21 +1,28 @@
 # Make sure specs run with the definitions from test.rb
 ENV['ROBOT_ENVIRONMENT'] = 'test'
 
+require 'simplecov'
 require 'coveralls'
 Coveralls.wear!
 
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+
 bootfile = File.expand_path(File.dirname(__FILE__) + '/../config/boot')
 require bootfile
 
 require 'pry'
 require 'rspec'
+require 'webmock/rspec'
 
 def setup_work_item(druid)
   @work_item=double("work_item")
   allow(@work_item).to receive_messages(:druid=>druid)
 end
 
-def setup_release_item(druid,obj_type,item_members=nil)
+def setup_release_item(druid,obj_type,item_members,collections)
```

Comment:
i think the reason it was set to nil before was to make item_members optional, but i don't think there is a way to make both optional without resorting to named parameters or a a hash as input...

but we could set both to nil by default, and if you need neither you can leave them both off, but if you need just collections (and not item_members), you'd be obligated to send a nil in for item_members (since it comes first in the list)